src/map.h

```diff
@@ -161,6 +161,105 @@ struct level_cache {
 };
 
 /**
+ * Base of the map class, currently rather underpopulated.
+ * Is NOT supposed to implement complex behaviors, only an interface to access submaps.
+ */
+class map_base
+{
+    protected:
+        map_base( int mapsize = MAPSIZE, bool zlev = false );
```

Comment:
Are those default values actually used anywhere, or does each call to the constructor supply explicit values? In the later case it would be good to remove the defaults as they serve no purpose.